rhino/src/main/java/org/mozilla/javascript/DecompilerFlag.java

```diff
@@ -0,0 +1,17 @@
+/* -*- Mode: java; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 4 -*-
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.javascript;
+
+public enum DecompilerFlag {
```

Comment:
Does it make sense to call this DecompilerFlag if there's no decompiling going on anymore?

Reply:
Do you have any good names?